site/package.json

```diff
@@ -13,7 +13,7 @@
     "dev:seed-db": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/seed-db",
     "generate-blocks-data": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/generate-blocks-data",
     "generate-sitemap": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/generate-sitemap",
-    "postinstall": "yarn generate-sitemap && yarn generate-blocks-data",
+    "postinstall": "patch-package && yarn generate-sitemap && yarn generate-blocks-data",
```

Comment:
Hmm just noticed that this call is inside a workspace. I think we should investigate why `ajv` is not hoisted and them move the script to the root `package.json`. That can be done later.